test/unittests/evm_eof_function_test.cpp

```diff
@@ -79,7 +79,7 @@ TEST_P(evm, callf_with_inputs_stack_size_1024)
         return;
 
     rev = EVMC_CANCUN;
-    const auto code = bytecode{"ef0001 010008 020002 0BFF 0004 030000 00 000003FF 03030004"_hex} +
```

Comment:
This seems to belong to the prev commit.

Reply:
Fixed